/usr/local/lib/python2.7/dist-packages/matplotlib/axes/_axes.py:545: UserWarning: No labelled objects found. Use label='...' kwarg on individual plots.
  warnings.warn("No labelled objects found. "
ClusterEmcee2_linear_mu_2step.py:165: RuntimeWarning: divide by zero encountered in log10
  mag = 22.5-2.5*np.log10(delta_f*mu['r'] + mu['r'])
('300', '400', 'normal')
Running object 300
0.0
{'i': 55, 'z': 155}
{'i': 0.6274861, 'r': 0.0, 'z': 1.6223955, 'g': 0.0}
Variable??
0
Non-Variable source in g
Variable??
0
Non-Variable source in r
Variable??
0.9429948596901492
Non-Variable source in i
Variable??
1.1121766071072368
Non-Variable source in z
Slope:       [1, 1, 1, 1]
Slope Error: [0.0, 0, 0.0, 0.0]
intercepts:  [0, 0, 0, 0]
Int Error:   [0.0, 0, 0.0, 0.0]
StDev:       [0.0, 0.0, 0.06274773, 0.123578146]
('ROW:', 300, 'Tau:', '2.4960711593407385', 'V:', '-1.7015284685832153')
Traceback (most recent call last):
  File "ClusterEmcee2_linear_mu_2step.py", line 512, in <module>
    perform_emcee_step2(time, flux_mod, err_mod, dMu_dict, scale_dict, color_sort_ones, ROW, mu)
  File "ClusterEmcee2_linear_mu_2step.py", line 341, in perform_emcee_step2
    sausageplot_step2(max_theta[0], time, flux, max_theta[1], 5, err**2, dMu_dict, scale_dict, color_sort_ones, ROW, fig)
  File "ClusterEmcee2_linear_mu_2step.py", line 173, in sausageplot_step2
    [xlim, ylim] = boundaries(time-57000, np.array(mag_arr))
  File "ClusterEmcee2_linear_mu_2step.py", line 296, in boundaries
    mjdmax = 100*np.ceil(np.max(mags)*.01)
  File "/home/thrush2/.local/lib/python2.7/site-packages/numpy/core/fromnumeric.py", line 2320, in amax
    out=out, **kwargs)
  File "/home/thrush2/.local/lib/python2.7/site-packages/numpy/core/_methods.py", line 26, in _amax
    return umr_maximum(a, axis, None, out, keepdims)
ValueError: zero-size array to reduction operation maximum which has no identity
